tests/run/C_argc_argv-to-java_args/src/main/scala/CArgcArgvToJavaArgs.scala

```diff
@@ -0,0 +1,10 @@
+object CArgcArgvToJavaArgs {
+  def main(args: Array[String]) {

```

Comment:
@Milyardo It's planned to be deprecated. It's not deprecated yet. We should probably avoid it in the future.